src/components/ImageSelector/relatedImageAnimation.css

```diff
@@ -0,0 +1,83 @@
+.relatedImageBase {
+    animation: relatedImageBaseAnimation 3s linear 0s normal forwards;
+}
+
+@keyframes relatedImageBaseAnimation {
+    0% {
```

Comment:
could you specify only 0% and 100% ? It should animate automatically between the 2

Reply:
Right, modification done